analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/Descriptors/RuleParameter.cs

```diff
@@ -19,31 +19,49 @@
  */
 
 using System.Diagnostics.CodeAnalysis;
+#if NETCOREAPP  // ToDo: Remove conditional compilation after change of target framework
+using System.Text.Json.Serialization;
+#endif
 using System.Xml;
 using System.Xml.Serialization;
+using SonarAnalyzer.Common;
 
-namespace SonarAnalyzer.RuleDescriptorGenerator
+namespace SonarAnalyzer.RuleDescriptorGenerator;
+
+[ExcludeFromCodeCoverage]
+public class RuleParameter
 {
-    [ExcludeFromCodeCoverage]
-    public class RuleParameter
+    // ToDo: Remove XML annotations
+    [XmlElement("key")]
+    public string Key { get; set; }
+
+    [XmlIgnore]
```

Comment:
As the Description was Ingored for Xml. Don't we also want to ignore it for Json?

Reply:
No, we want description. This is ignored for XML serialization, because it is serialized via the `DescriptionCDataSection` below to XML. 

For JSON, this original `Description` is correct. The `DescriptionCDataSection` is `JsonIgnore`d and will be deleted later this week.